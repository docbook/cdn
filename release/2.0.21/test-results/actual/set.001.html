<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: set.001</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.21" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="set"><header class="set-titlepage"><h1>Unit Test: set.001</h1><div class="author"><h3><span class="personname"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span></h3></div><p class="CVS">$Id: set.001.xml 4056 2004-11-23 13:28:50Z nwalsh $</p></header><div class="content"><article id="R.1.2" class="book"><header class="book-titlepage"><h1>First Book</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.2.2">1. Chapter</a></span></li></ul></div></div><div class="content"><article id="R.1.2.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><div class="content"><p>Some test data</p></div></article></div></article><article id="R.1.3" class="book"><header class="book-titlepage"><h1>Second Book</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.3.2">1. Chapter</a></span></li></ul></div></div><div class="content"><article id="R.1.3.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><div class="content"><p>Some test data</p></div></article></div></article><article id="R.1.4" class="book"><header class="book-titlepage"><h1>Third Book</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.4.2">1. Chapter</a></span></li></ul></div></div><div class="content"><article id="R.1.4.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><div class="content"><p>Some test data</p></div></article></div></article></div></article><script src="../../resources/js/prism.js"></script></body></html>