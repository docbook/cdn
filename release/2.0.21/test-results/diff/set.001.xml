<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: set.001</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="set">
         <header deltaxml:deltaV2="A=B" class="set-titlepage">
            <h1>Unit Test: set.001</h1>
            <div class="author">
               <h3>
                  <span class="personname">
                     <span class="firstname">Norman</span>
                     <span class="surname">Walsh</span>
                  </span>
               </h3>
            </div>
            <p class="CVS">$Id: set.001.xml 4056 2004-11-23 13:28:50Z nwalsh $</p>
         </header>
         <article deltaxml:deltaV2="A" id="R.1.2" class="book">
            <header class="book-titlepage">
               <h1>First Book</h1>
            </header>
            <div class="lists-of-titles">
               <div class="toc">
                  <p>
                     <b>Table of Contents</b>
                  </p>
                  <ul class="toc">
                     <li>
                        <span>
                           <a href="#R.1.2.2">1. Chapter</a>
                        </span>
                     </li>
                  </ul>
               </div>
            </div>
            <article id="R.1.2.2" class="chapter">
               <header class="chapter-titlepage">
                  <h2>Chapter 1. Chapter</h2>
               </header>
               <p>Some test data</p>
            </article>
         </article><div deltaxml:deltaV2="B" class="content">
            <article id="R.1.2" class="book">
               <header class="book-titlepage">
                  <h1>First Book</h1>
               </header>
               <div class="lists-of-titles">
                  <div class="toc">
                     <p>
                        <b>Table of Contents</b>
                     </p>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#R.1.2.2">1. Chapter</a>
                           </span>
                        </li>
                     </ul>
                  </div>
               </div>
               <div class="content">
                  <article id="R.1.2.2" class="chapter">
                     <header class="chapter-titlepage">
                        <h2>Chapter 1. Chapter</h2>
                     </header>
                     <div class="content">
                        <p>Some test data</p>
                     </div>
                  </article>
               </div>
            </article>
            <article id="R.1.3" class="book">
               <header class="book-titlepage">
                  <h1>Second Book</h1>
               </header>
               <div class="lists-of-titles">
                  <div class="toc">
                     <p>
                        <b>Table of Contents</b>
                     </p>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#R.1.3.2">1. Chapter</a>
                           </span>
                        </li>
                     </ul>
                  </div>
               </div>
               <div class="content">
                  <article id="R.1.3.2" class="chapter">
                     <header class="chapter-titlepage">
                        <h2>Chapter 1. Chapter</h2>
                     </header>
                     <div class="content">
                        <p>Some test data</p>
                     </div>
                  </article>
               </div>
            </article>
            <article id="R.1.4" class="book">
               <header class="book-titlepage">
                  <h1>Third Book</h1>
               </header>
               <div class="lists-of-titles">
                  <div class="toc">
                     <p>
                        <b>Table of Contents</b>
                     </p>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#R.1.4.2">1. Chapter</a>
                           </span>
                        </li>
                     </ul>
                  </div>
               </div>
               <div class="content">
                  <article id="R.1.4.2" class="chapter">
                     <header class="chapter-titlepage">
                        <h2>Chapter 1. Chapter</h2>
                     </header>
                     <div class="content">
                        <p>Some test data</p>
                     </div>
                  </article>
               </div>
            </article>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><article deltaxml:deltaV2="A" id="R.1.3" class="book">
            <header class="book-titlepage">
               <h1>Second Book</h1>
            </header>
            <div class="lists-of-titles">
               <div class="toc">
                  <p>
                     <b>Table of Contents</b>
                  </p>
                  <ul class="toc">
                     <li>
                        <span>
                           <a href="#R.1.3.2">1. Chapter</a>
                        </span>
                     </li>
                  </ul>
               </div>
            </div>
            <article id="R.1.3.2" class="chapter">
               <header class="chapter-titlepage">
                  <h2>Chapter 1. Chapter</h2>
               </header>
               <p>Some test data</p>
            </article>
         </article><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><article deltaxml:deltaV2="A" id="R.1.4" class="book">
            <header class="book-titlepage">
               <h1>Third Book</h1>
            </header>
            <div class="lists-of-titles">
               <div class="toc">
                  <p>
                     <b>Table of Contents</b>
                  </p>
                  <ul class="toc">
                     <li>
                        <span>
                           <a href="#R.1.4.2">1. Chapter</a>
                        </span>
                     </li>
                  </ul>
               </div>
            </div>
            <article id="R.1.4.2" class="chapter">
               <header class="chapter-titlepage">
                  <h2>Chapter 1. Chapter</h2>
               </header>
               <p>Some test data</p>
            </article>
         </article>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>