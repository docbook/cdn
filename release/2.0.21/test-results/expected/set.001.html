<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: set.001</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="set"><header class="set-titlepage"><h1>Unit Test: set.001</h1><div class="author"><h3><span class="personname"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span></h3></div><p class="CVS">$Id: set.001.xml 4056 2004-11-23 13:28:50Z nwalsh $</p></header><article id="R.1.2" class="book"><header class="book-titlepage"><h1>First Book</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.2.2">1. Chapter</a></span></li></ul></div></div><article id="R.1.2.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><p>Some test data</p></article></article><article id="R.1.3" class="book"><header class="book-titlepage"><h1>Second Book</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.3.2">1. Chapter</a></span></li></ul></div></div><article id="R.1.3.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><p>Some test data</p></article></article><article id="R.1.4" class="book"><header class="book-titlepage"><h1>Third Book</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.4.2">1. Chapter</a></span></li></ul></div></div><article id="R.1.4.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><p>Some test data</p></article></article></article><script src="../../resources/js/prism.js"></script></body></html>